File: src/Lession5/Ex1.java
package Lession5;

public class Ex1 {
    static void main() {
        int[] Array = new int[] {3, -7, 0, 9, -2, -10, 6, 1, -5, 8, -3, 4, -9, 2, 7, -1, 10, -6, 5, 0};

        int min_index = 0;
        int max_index = 0;
        int min = Array[0];
        int max = Array[0];

        for (int i = 1; i < Array.length; i++) {
            if (Array[i] > max) {
                max = Array[i];
                max_index = i;
            } else if (Array[i] < min) {
                min = Array[i];
                min_index = i;
            }
        }

        System.out.println("Min: " + min + " at index " + min_index);
        System.out.println("Max: " + max + " at index " + max_index);
    }
}

File: src/Lession5/Ex1.kt
package Lession5

fun main() {
    val array = intArrayOf(-29, 14, -4, 0, 17, -16, 3, -11, 8, -2, 19, -7, 5, -20, 12, -3, 6, -14, 1, 10)

    var max = array[0];
    var min = array[0];
    var max_index = 0;
    var min_index = 0;

    for (i in 1 until array.size) {
        if (array[i] > max) {
            max = array[i];
            max_index = i;
        } else if (array[i] < min) {
            min = array[i];
            min_index = i;
        }
    }

    println("Max: $max at index $max_index")
    println("Min: $min at index $min_index")
}
